packages/i18n/scripts/compile_data.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+export NODE_ENV="production"
```

Comment:
Since the scripts rely on the constants package, babel needs to know the environment, so we're setting it explicitly.